third_party/terraform/resources/resource_container_cluster.go.erb

```diff
@@ -3567,6 +3601,18 @@ func flattenMaintenancePolicy(mp *containerBeta.MaintenancePolicy) []map[string]
 	if mp == nil || mp.Window == nil {
 		return nil
 	}
+
+	exclusionWindows := []map[string]interface{}{}
+	if mp.Window.MaintenanceExclusions != nil {
+		for wname, tw := range mp.Window.MaintenanceExclusions {
```

Comment:
What does `tw` represent here? Maybe rename to `window` or similar?

Reply:
tw - timewindow. I can change it to window